### DISCUSSION TITLE
YunoHost Package Install Failure

### DISCUSSION THREAD

**Discussion by N-Upchurch** [apache/couchdb#3677] [Q&A] [answered]
Per [this issue](https://github.com/YunoHost-Apps/couchdb_ynh/issues/11), the YunoHost CouchDB package is failing to install. 

It looks like the culprit is maybe an incorrect repo, judging by the warnings I received prior to failure:
```
2021-07-09 21:37:25,036: WARNING - E: Failed to fetch https://apache.bintray.com/couchdb-deb/dists/buster/InRelease  403  Forbidden [IP: 35.157.24.53 443]
2021-07-09 21:37:25,038: WARNING - E: The repository 'https://apache.bintray.com/couchdb-deb buster InRelease' is not signed.
```
Could I just swap that URL in that line (line 100) of the [installation script](https://github.com/YunoHost-Apps/couchdb_ynh/blob/master/scripts/install) to the new repo? (if so - what should I swap it to?) I'm not super knowledgeable on this, but if someone could steer me in the right direction, I may be able to attempt to resolve it and submit a pull request.

**Comment by wohali** [comment, marked as answer]
We know nothing about YunoHost. You'd do best to work with those folks to sort out the installation process.

Here is the new process for installing CouchDB on Linux from packages. Note that our  package signing and signing key has changed as well.

https://docs.couchdb.org/en/latest/install/unix.html

**Reply by N-Upchurch** [reply]
Thanks for your reply. It looks like they updated the testing branch overnight to reflect the changes.
